easybuild/main.py

```diff
@@ -409,6 +410,8 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     # don't try and tweak anything if easyconfigs were generated, since building a full dep graph will fail
     # if easyconfig files for the dependencies are not available
     if try_to_generate and build_specs and not generated_ecs:
+        if options.dump_tweaked_easyconfig:
+            tweaked_ecs_paths = [os.getcwd(), os.getcwd()]
```

Comment:
I was about to consider a PR related to this, I have a few problems with this approach. It's greedy, it will copy over all the tweaked easyconfigs and place them indiscriminately in the current directory. This is not what happens behind the scenes though, and that is why we use two paths. The first is prepended to the robot path, the second appended, by setting them to be equal you are giving precedence to the tweaked easyconfigs and this is not what you want.
It's also pretty complicated for a user to reproduce that search hierarchy (you *must* explicitly set the robot path and the config value needs to be sandwiched in the middle), you'd need to advise them how to do that.
What I was considering was doing this copy once the dependencies have been resolved. At that point you can check for `tweaked_easyconfigs` and `tweaked_dep_easyconfigs` in the paths of the final ec's and only dump those. This would remove the need to worry about setting the robot path.

Reply:
I trust you on that. To be honest, this has been opened for so long, I don't even know if the problem I was trying to address is still there.

Reply:
I totally see the use case, you want easy access to the tweaked easyconfigs, there are a couple of quirks to that though.

Reply:
I think I have something that works in #3332

Reply:
How does it compare to @boegel's https://github.com/easybuilders/easybuild-framework/pull/2665 ?

Reply:
I think he better answer that...

Reply:
You can just close this current PR if you want.